### ISSUE TITLE
no-duplicate-tags.js should report one error per tag duplicated

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#173]
Hi!

I think it should be reported one error for each duplication. Then if there are three tags with the same name in a certain node, should be reported as error the second and third occurrence.

I mean
``` gherkin
Feature:

@bar @foo @bar @bar
Scenario:
  Then step
```
should report two errors.
